docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,602 @@
+* https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html?extIdCarryOver=true&sc_cid=701f2000001Css5AAC[Ansible 2.9^] or later is installed.
+* Some additional dependencies required by this collection are installed. To install these, enter the following commands:
++
+[source,shell]
+----
+$ pip install rhoas-sdks
+$ pip install python-dotenv
+----
+NOTE: Use the `pip install rhoas-sdks --force reinstall` command to ensure that the most up to date versions of the SDKs are installed.
++
+
+* (Optional) This collection works best when used in a Python virtual environment. To create and activate a Python virtual environment, enter the following command:
++
+[source,shell]
+----
+$ python3 -m venv rhoas/
+. rhoas/bin/activate
+----
++
+
+.Procedure
+
+. Open a terminal and enter the following command:
++
+[source,shell]
+----
+$ ansible-galaxy collection install rhoas.rhoas
+----
++
+By default, the `ansible-galaxy collection install` command uses https://galaxy.ansible.com as the Galaxy server URL. The URL is specified as the value of the `GALAXY_SERVER` parameter in the `ansible.cfg` configuration file.
+. Once installed, you can reference a collection content by its fully qualified collection name (FQCN). For example the `create_kafka` command in a task needs to be the FQCN of the module for collection development, for example
+`rhoas.rhoas.create_kafka`.
+
+[id="proc-using-rhoas-ansible-collection_{context}"]
+
+=== Using the RHOAS Ansible collection
+
+.Prerequisites
+* You have a Red Hat account.
+* You have an offline token used to authenticate the Ansible modules with the {product-long-rhoas} API.
+
+[NOTE]
+The offline token is a refresh token with no expiry and can be used by non-interactive processes to provide an access token for {product-long-rhoas} to authenticate the user. The token is an OpenShift offline token and you can find it at https://cloud.redhat.com/openshift/token.
+
+* The collection requires the following two variables to work:
+API_BASE_HOST:: The base host for the API. This is the base URL for the API. For example, https://api.openshift.com.
+SSO_BASE_HOST:: The base host for the SSO. This is the base URL for the SSO. For example, https://sso.redhat.com/auth/realms/redhat-external.
+You can define the environment variables shown in an environment variable (`.env`) file and save the file in the root directory. If you don't explicitly define these environment variables, the collection uses the URLs shown in the examples.
+
+.Procedure
+
+. (Optional) To find documentation for each module, open a terminal window, enter `ansible-doc rhoas.rhoas.<module_name>` and replace <module_name> with the name of the module in the terminal. The following example outputs the documentation for the `create_kafka` module. The documentation shows required parameters (denoted by `=`) and optional parameters (denoted by `-`) for the module, as well as examples of how to use the module and the output.
```

Comment:
A couple of comments:

- I think this note was better in it's other location, up near line 164. I'm not sure I'd begin a procedure with two optional steps.
- Format the replaceable <module_name> in italics by enclosing in underscores:

```
(Optional) To find documentation for each module, open a terminal window, enter `ansible-doc rhoas.rhoas._<module_name>_` and replace _<module_name>_ with the name of the module in the terminal.
```